pkg/cli/subcommands/check/command.go

```diff
@@ -182,8 +182,8 @@ func run(log log.Component, config config.Component, sysprobeconfig sysprobeconf
 
 	// Always disable SBOM collection in `check` command to avoid BoltDB flock issue
 	// and consuming CPU & Memory for asynchronous scans that would not be shown in `agent check` output.
-	pkgconfig.Datadog.Set("sbom.enabled", "false")
-	pkgconfig.Datadog.Set("container_image_collection.sbom.enabled", "false")
+	pkgconfig.Datadog.Set("sbom.host.enabled", "false")
```

Comment:
This is a breaking change. Is it an issue?

Reply:
It’s indeed a breaking change. But it affects a beta feature which isn’t publicly documented.
We share with beta customers a document explaining how to enable the feature depending on the agent version.